recipes/flit/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.9" %}
+
+package:
+  name: flit
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/takluyver/flit.git
+  git_tag: {{ version }}
```

Comment:
Happy to, but if this is the preferred mechanism, you may want to update the example recipe.